ynput/ayon-deadline
# Houdini: Set frames if cache instance has multiple frames

## Changelog Description
resolve #61 
This PR is based on the collected `frames` at an early stage of cache publishing. 
and if `frames` is a string then it's one file and if `frames` is a list then it includes multiple files.

## Testing notes:
1. Create two alembic instances, use `$F4` in one of them.
2. Submit to Farm using the publisher tool
3. The instance with explicit frames (`$F4`) should have frames in the DL job.
    ![image](https://github.com/user-attachments/assets/1288d016-4620-4d2c-b199-e438c78f8d01)
    The other one should have the default value 0 (which means it exports one frame.)
    ![image](https://github.com/user-attachments/assets/02dc88c4-00bc-42d1-8625-f4b0219f0853)
4. Other types of cache instances should work as usual.
> [!Note]
> Test note number 3 is also true for other cache types like Bgeo and VDB for example.
> However, it won't make sense to remove the `$F4` from them because each frame will override the previous frame.
